src/acl.js

```diff
@@ -0,0 +1,53 @@
+class ACL {
```

Comment:
Maybe we could call it `IntegrityChecker` to avoid confusion around orbit-db and ipfs-log ACL